README.md

```diff
@@ -0,0 +1,98 @@
+![Github CI Status](https://github.com/oasis-roles/index_href/workflows/Role%20CI/badge.svg)
+
+index_href
+===========
+
+Provides a lookup plugin that can parse links in a web directory index
```

Comment:
When it comes to more general "utility" roles/plugins like this, I prefer the objective "this is what this does" message instead of the subjective "this is what this is for". It might be useful to provide this and other examples, but I'd prefer to discuss that in a separate enhancement issues independent from this pull request unless you think it's critical to the role.

Reply:
#2